dartagnan/src/main/java/com/dat3m/dartagnan/Dartagnan.java

```diff
@@ -397,4 +397,18 @@ private static void printWarningIfThreadStartFailed(Program p, EncodingContext e
             }
         }
     }
+
+    private static void printSpecification(StringBuilder sb, Program program) {
+        sb.append("Condition ").append(program.getSpecificationType()).append(" ");
```

Comment:
print as lower case to match the previous behaviour and the litmus input format.